packages/@aws-cdk/aws-appconfig-alpha/lib/extension.ts

```diff
@@ -290,21 +290,29 @@ export interface ExtensionAttributes {
 
   /**
    * The Amazon Resource Name (ARN) of the extension.
+   *
+   * @default - The ARN is generated.
```

Comment:
nit: wish this was a little more descriptive.

Reply:
Changed to `The extension ARN is generated`